src/view/geometry/Transform.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2012-2015 S-Core Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file Introduction
+ * @since 1.0.0
+ * @author youngd.hwang@samsung.com
+ */
+
+define([
+    'external/genetic/genetic',
+    'graphite/view/geometry/Point',
+    'graphite/base/Base'
+], function (
+    genetic,
+    Point,
+    Base
+) {
+    'use strict';
+
+    /**
+     * A Transform.
```

Comment:
Please add introduction for this file.
"A Transform class which provides support for transformations of scaling, rotation and translation." may be.